src/css/base.scss

```diff
@@ -3,26 +3,59 @@ $primary-background: radial-gradient(circle, rgba(63,94,251,1) 0%, rgba(252,70,1
 ;
 
 body {
-
+ margin: 0;
+ font-family: "RocknRoll One"
 }
 
 header {
   display: flex;
   justify-content: space-between;
+  background-color: #BAD4FA;
+  width: 100%;
+  .user {
+    margin-right: 10px;
```

Comment:
Margin is used a lot throughout this file. Consider using a mixin or slim it down by using the margin: up right down left; syntax rather than addressing each margin direction separately.